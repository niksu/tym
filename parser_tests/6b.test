%v(a).
%v(b).
%v(c).
%e(a,a).
%e(b,b).
%e(a,b).
%e(b,c).
d(X, X).
e(X,Y) :- d(X, Z), d(Z, Y).
%d(X, X).
