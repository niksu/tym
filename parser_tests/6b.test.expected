input contents |%v(a).
%v(b).
%v(c).
%e(a,a).
%e(b,b).
%e(a,b).
%e(b,c).
d(X, X).
e(X,Y) :- d(X, Z), d(Z, Y).
%d(X, X).
|
(no query given)
stringed file contents (size=78, remaining=2922)
|predicate_d(X, X).
predicate_e(X, Y) :- predicate_d(X, Z), predicate_d(Z, Y).|
stringed file contents strlen=77
